sky/serve/serve_state.py

```diff
@@ -43,14 +44,22 @@ def create_table(cursor: 'sqlite3.Cursor', conn: 'sqlite3.Connection') -> None:
         replica_id INTEGER,
         replica_info BLOB,
         PRIMARY KEY (service_name, replica_id))""")
-
+    cursor.execute("""\
+        CREATE TABLE IF NOT EXISTS versions (
+            service_name TEXT,
+            version INTEGER,
+            spec BLOB,
+            yaml STR,
```

Comment:
do we really need both yaml and spec? iiuc the yaml includes the spec?

Reply:
let's keep only yaml